### DISCUSSION TITLE
pull requests merge to main (production) branch but show up as the merge branch

### DISCUSSION THREAD

**Discussion by codeforcesmeme** [vercel/vercel#5031] [Help]
I have a problem when merging a PR from branch 'A' into my main branch 'main'.
It shows up in both `https://github.com/username/repo/deployments` and vercel dashboard as my production branch.
<details>
<summary>
Like this
</summary>

![this](https://i.imgur.com/LduHe8e.png)

</details>

<details>
<summary>
and this
</summary>

![this](https://i.imgur.com/GVna1ef.png)

</details>

But I want main to show up as my production branch, because we basically don't ever push to it without a PR, so 'production' in dashboards will always be a branch that often gets deleted once merged.

**Comment by MaxLeiter** [comment]
Hey @codeforcesmeme, have you tried setting your production branch in the Vercel dashboard? You can find out how here: https://vercel.com/docs/git-integrations#customizing-the-production-branch

**Reply by codeforcesmeme** [reply]
It is set to be master.
It sort of lazy load the build by adding the production domain to the old deployment instance with the old branch name and it leaves my 'preview deployments' section into the 'production deployments' even though it was merged to master but it's the same src of the last preview, I think this is what causes this.
